specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/definitions.json

```diff
@@ -5151,6 +5151,163 @@
         }
       },
       "description": "Subscription keys."
+    },
+    "GatewayCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GatewayContract"
+          },
+          "description": "Page values.",
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any.",
+          "readOnly": true
+        }
+      },
+      "description": "Paged Gateway list representation."
+    },
+    "GatewayContract": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GatewayContractProperties",
+          "description": "Gateway details."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./apimanagement.json#/definitions/Resource"
+        }
+      ],
+      "description": "Gateway details."
+    },
+    "GatewayContractProperties": {
+      "properties": {
+        "region": {
```

Comment:
>"region" [](start = 8, length = 8)

what is the difference between this property and a normal top-level "location" property?  What does this region impact?

Reply:
Gateway resource represents containerized API Management data plane that can be self-hosted by customer on compute of his choice (on-prem, in azure or other cloud) and connected to managed API Management Service. On Service can govern multiple Gateways.
top-level location corresponds to the Service.
Gateway-level location corresponds to Gateway.